rules/sqlperf-use-and-instead-of-or/rule.md

```diff
@@ -0,0 +1,35 @@
+---
+seoDescription: Do you know that AND is much more efficient than OR in a WHERE clause?
+type: rule
+archivedreason:
+title: Do you know that AND is much more efficient than OR in a WHERE clause?
+guid: b4be79d9-6bf8-4b31-b222-30faff037d98
+uri: sqlperf-use-and-instead-of-or
+created: 2024-07-23T07:26:54.0000000Z
+authors:
+  - title: Bryden Oliver
+    url: https://ssw.com.au/people/bryden-oliver
+related:
+  - sqlperf-reduce-table-size
+  - sqlperf-select-required-columns
+  - sqlperf-verify-indexes-used
+  - sqlperf-avoid-implicit-type-conversions
+  - sqlperf-avoid-looping
+  - sqlperf-minimise-large-writes
+---
+
+Did you know that using AND is much more efficient than using OR? Find out why.
```

Comment:
Hi @brydeno 

I don't think we should repeat (kind of) the title here... I would make this a summary of the main point of the rule

Reply:
adjusted.